packages/dashboard/src/components/Wizard/Wizard.Base.tsx

```diff
@@ -19,33 +20,41 @@ export class WizardBase extends React.Component<IWizardProps, {}> {
       throw new Error('Wizard must have atleast one step.');
     }
 
-    const classNames = getClassNames(this.props.styles!, { theme: this.props.theme! });
-
     // if the step to render is already passed in, use that
     const wizardStepProps = this.props.stepToShow ? this.props.stepToShow : getStepToShow(this.props);
 
+    const classNames = getClassNames(this.props.styles!, {
+      theme: this.props.theme!,
+      isSubStep: wizardStepProps.isSubStep!,
+      isFirstSubStep: wizardStepProps.isFirstSubStep!
+    });
+
     return (
       <div className={classNames.wizardContentNavContainer}>
         <div className={classNames.subwayNavSection}>
           <SubwayNav steps={steps} wizardComplete={this.props.wizardComplete} />
         </div>
-        <div className={classNames.contentSection}>
-          {this._onRenderContentTitle(wizardStepProps.wizardContent)}
-          {this._onRenderContent(wizardStepProps.wizardContent)}
-        </div>
+        {this._onRenderContentSection(wizardStepProps, classNames)}
```

Comment:
When I look at this I'm not sure we have enough complexity to break it out into methods. We could just conditionally render inline then we don't have anything to pass. If there was more complex logic I think it would be worth it but I think it makes it harder to understand broken up.